src/uglify/index.js

```diff
@@ -1,10 +1,9 @@
 import os from 'os';
+import cacache from 'cacache';
 import findCacheDir from 'find-cache-dir';
```

Comment:
@Kovensky It is a `{Function}` which 'contains' the value =>

1. Directly pass as an argument with 'higher order' => `const cache = (data, cacheDir(options)) => cache`
2. Assign it to a scope/block/'namespace' instead of an 'redundant' variable => `this.cacheDir = cacheDir(options) || const cache = { dir: cacheDir(options) }` 

Anyways just a nitpick 🐦 😛